### PR TITLE
Scip v804

### PR DESCRIPTION
**Pull request by ju-manns**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Review by AntoinePrv — APPROVED** [review]

**Comment by matbesancon** [comment]
@AntoinePrv do you know what's going on in the installation step here:

```
2023-08-03T15:52:18.6189668Z -- The C compiler identification is GNU 12.3.0
2023-08-03T15:52:18.7310082Z -- The CXX compiler identification is GNU 12.3.0
2023-08-03T15:52:18.7519350Z -- Detecting C compiler ABI info
2023-08-03T15:52:18.8561270Z -- Detecting C compiler ABI info - done
2023-08-03T15:52:18.8812359Z -- Check for working C compiler: $PREFIX/bin/x86_64-conda-linux-gnu-cc - skipped
2023-08-03T15:52:18.8821830Z -- Detecting C compile features
2023-08-03T15:52:18.8833128Z -- Detecting C compile features - done
2023-08-03T15:52:18.8905589Z -- Detecting CXX compiler ABI info
2023-08-03T15:52:18.9930618Z -- Detecting CXX compiler ABI info - done
2023-08-03T15:52:19.0195041Z -- Check for working CXX compiler: $PREFIX/bin/x86_64-conda-linux-gnu-c++ - skipped
2023-08-03T15:52:19.0200098Z -- Detecting CXX compile features
2023-08-03T15:52:19.0210550Z -- Detecting CXX compile features - done
2023-08-03T15:52:19.0286697Z CMake Error at CMakeLists.txt:9 (find_package):
2023-08-03T15:52:19.0287328Z   By not providing "FindSCIP.cmake" in CMAKE_MODULE_PATH this project has
2023-08-03T15:52:19.0289015Z   asked CMake to find a package configuration file provided by "SCIP", but
2023-08-03T15:52:19.0289486Z   CMake did not find one.
2023-08-03T15:52:19.0289634Z 
2023-08-03T15:52:19.0289858Z   Could not find a package configuration file provided by "SCIP" with any of
2023-08-03T15:52:19.0290234Z   the following names:
2023-08-03T15:52:19.0290324Z 
2023-08-03T15:52:19.0290479Z     SCIPConfig.cmake
2023-08-03T15:52:19.0290750Z     scip-config.cmake
2023-08-03T15:52:19.0290839Z 
2023-08-03T15:52:19.0291029Z   Add the installation prefix of "SCIP" to CMAKE_PREFIX_PATH or set
2023-08-03T15:52:19.0291272Z   "SCIP_DIR" to a directory containing one of the above files.  If "SCIP"
2023-08-03T15:52:19.0291787Z   provides a separate development package or SDK, be sure it has been
2023-08-03T15:52:19.0291984Z   installed.
2023-08-03T15:52:19.0292067Z 
2023-08-03T15:52:19.0292136Z 
2023-08-03T15:52:19.0292373Z -- Configuring incomplete, errors occurred!
2023-08-03T15:52:19.6368467Z Tests failed for scip-8.0.4-h4a32fe0_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
```

not sure if some particular configuration / step is expected?

**Comment by AntoinePrv** [comment]
Looks like a test failing. The test example cannot find the SCIP CMake config files.

Looking at the installation, we have:
```
-- Installing: $PREFIX/lib64/libscip.so.8.0.4.0
-- Installing: $PREFIX/lib64/libscip.so.8.0
-- Installing: $PREFIX/lib64/libscip.so
-- Installing: $PREFIX/lib64/cmake/scip/scip-targets.cmake
-- Installing: $PREFIX/lib64/cmake/scip/scip-targets-release.cmake
-- Installing: $PREFIX/lib64/cmake/scip/scip-config.cmake
```
which is supposed to be under `$PREFIX/lib` (not `lib64`) in Conda. This can be controlled by `CMAKE_INSTALL_LIBDIR`. This is properly set in Conda-Froge's `CMAKE_ARGS` environment variable.
You could try adding `$CMAKE_ARGS` (no quote) to the CLI arguments [here](https://github.com/conda-forge/scipoptsuite-feedstock/blob/b634cde310f82c0d55ca8871e39bc2573766627e/recipe/build.sh#L18). And similarily `%CMAKE_ARGS%` on Windows.

File: recipe/build.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ cmake -B scipoptsuite-build -S "${SRC_DIR}/scipoptsuite" \
       -D READLINE=OFF \
       -D SYM=bliss \
       -D EXPRINT=cppad \
-      -D CLIQUER=ON
+      -D CLIQUER=ON \
+      -D CMAKE_INSTALL_LIBDIR=$PREFIX/lib
```
Comment (AntoinePrv) [inline recipe/build.sh]:
```suggestion
      -D CMAKE_INSTALL_LIBDIR=lib
```
We use relative, the absolute part is handled by `CMAKE_INSTALL_PREFIX` or `--prefix`, but really I would recommend using `${CMAKE_ARGS}` as it contains (and evolves) other conda-forge relevant variables.

**Review by AntoinePrv — CHANGES_REQUESTED** [review]

**Comment by AntoinePrv** [comment]
Gonna try out the PR locally

**Review by AntoinePrv — APPROVED** [review]
All good now

**Comment by matbesancon** [comment]
thanks @AntoinePrv !

**Comment by ju-manns** [comment]
Thank you @AntoinePrv  :)

**Pull request merged**